scripts/rebuild-testbench.sh

```diff
@@ -34,22 +49,89 @@ export_CC_with_afl()
     export CC=${SOF_AFL}
 }
 
+setup_xtensa_tools_build()
+{
+    BUILD_TYPE=xt
+    BUILD_TARGET=
+    BUILD_DIR_NAME=build_xt_testbench
+
+    # check needed environment variables
+    test -n "${XTENSA_TOOLS_ROOT}" || die "XTENSA_TOOLS_ROOT need to be set.\n"
+
+    # Get compiler version for platform
+    source "$SCRIPT_DIR/set_xtensa_params.sh" "$BUILD_PLATFORM"
+
+    test -n "${XTENSA_TOOLS_VERSION}" ||
+	die "Illegal platform $BUILD_PLATFORM, no XTENSA_TOOLS_VERSION found.\n"
+    test -n "${XTENSA_CORE}" ||
+	die "Illegal platform $BUILD_PLATFORM, no XTENSA_CORE found.\n"
```

Comment:
indent

Reply:
Sorry, I have no idea what's wrong with indent. Help?

Reply:
Spaces on one line and tabs on the next feels fishy, even for shell scripts.

Reply:
It's unfortunate that we have some files with tabs and other files with spaces in the same project but I agree 100% with @paulstelian97 : let's please not cross the line where we have a mix of tabs and spaces _in the same file_!

So @singalsu please find the shortcut in your editor that lets you quickly switch between tabs and spaces like the rest of us :-)


PS: I can't resist sorry: this sort of time-consuming discussion is exactly why tabs suck. Off-topic.

Reply:
The file is already without my patch a mix of tabs and spaces. I can convert them all to spaces then.

Reply:
I checked and there are only 5 tabs right now. Feel free to convert them to spaces or to leave them alone, as long as you don't add any it's fine by me.